administrator/components/com_admin/script.php

```diff
@@ -2213,6 +2213,11 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/templates/system/incompatible.html,',
             '/media/plg_system_compat',
             '/media/plg_editors_tinymce/js/plugins/highlighter',
+            // From 5.0.0-alpha4 to 5.0.0-alpha5
+            'administrator/language/en-GB/plg_twofactorauth_totp.ini',
+            'administrator/language/en-GB/plg_twofactorauth_totp.sys.ini',
+            'administrator/language/en-GB/plg_twofactorauth_yubikey.ini',
+            'administrator/language/en-GB/plg_twofactorauth_yubikey.sys.ini',
```

Comment:
```suggestion
            // From 5.0.0-alpha4 to 5.0.0-beta1
            '/administrator/language/en-GB/plg_twofactorauth_totp.ini',
            '/administrator/language/en-GB/plg_twofactorauth_totp.sys.ini',
            '/administrator/language/en-GB/plg_twofactorauth_yubikey.ini',
            '/administrator/language/en-GB/plg_twofactorauth_yubikey.sys.ini',
```
next version is beta1?